lib/controllers/items.js

```diff
@@ -1,5 +1,42 @@
 const { Router } = require('express');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+const Item = require('../models/Item');
 
-module.exports = Router();
+module.exports = Router()
+  .get('/', authenticate, async (req, res, next) => {
+    try {
+      const item = await Item.getAll(req.user.id);
+      res.json(item);
+    } catch (error) {
+      next(error);
+    }
+  })
 
+  .post('/', authenticate, async (req, res, next) => {
+    try {
+      const item = await Item.insert({ ...req.body, user_id: req.user.id });
+      res.json(item);
+    } catch (error) {
+      next(error);
+    }
+  })
+
+  .put('/:id', [authenticate, authorize], async (req, res, next) => {
+    try {
+      const item = await Item.updateById(req.params.id, req.body);
+      res.json(item);
+    } catch (error) {
+      next(error);
+    }
+  })
+
+  .delete('/:id', async (req, res, next) => {
+    try {
+      const data = await Item.delete(req.params.id);
+      res.json(data);
+    } catch (error) {
+      next(error);
+    }
+  });
```

Comment:
It looks like we are missing our authenticate and authorize middleware on this route.